cores/arduino/wiring_digital_extra.h

```diff
@@ -40,7 +40,7 @@ void digitalToggle(pin_size_t ulPin);
 *****************************************************************************/
 typedef enum afio_debug_cfg {
     /** Full Serial Wire and JTAG debug */
-    AFIO_DEBUG_FULL_SWJ          = ((uint32_t)0x00300000U | (PCF0_SWJ_CFG(0) >> 16)),
+    AFIO_DEBUG_FULL_SWJ          = ((uint32_t)0x00300000U | (AFIO_PCF0_SWJ_CFG >> 16)),
```

Comment:
Does this really want to change for all devices? It feels like instead we might want to define some compat macros that are ifdefed to the F10x so we don't need to make this change inside the arduino core?

Reply:
fixed